app/feature/feature-chat/src/main/kotlin/com/hedvig/android/feature/chat/ui/ChatLoadedScreen.kt

```diff
@@ -142,6 +142,10 @@ internal fun ChatLoadedScreen(
           .clearFocusOnTap()
           .nestedScroll(topAppBarScrollBehavior.nestedScrollConnection),
       )
+      uiState.bannerText?.let {
+        Divider(Modifier.fillMaxWidth())
+        ChatBanner(text = it)
```

Comment:
```suggestion
        ChatBanner(text = it, Modifier.fillMaxWidth())
```
Could it be that this also needs to be told to go full width? How does it function if the content passed into it only takes some part of the width available. Does it expand already due to some other thing inside ChatBanner perhaps?